app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    if len(command) != 3 or command[0] != "mv":
+        raise ValueError(f"Invalid command: {command}")
+
+    source_path = command[1]
+    destination_path = command[2]
+
+    if not os.path.isfile(source_path):
+        raise FileNotFoundError(f"Source file does not exist: {source_path}")
```

Comment:
Ensure that the error message uses consistent double quotes as per the code style checklist.